README.md

```diff
@@ -401,19 +404,55 @@ In this section, we'll cover the security requirements and features of Online Bo
 - **DELETE:** `/api/categories/{id}` (Delete a category)
 - **PATCH:** `/api/orders/{id}` (Update order status)
 
-## Running the Application with Docker
-In this section, you will find out how the process of running application using Docker.
+## Running the Application 
+
+In this section, you will find out how the process of running application.
 - [Back to application content](#content)
+
+####
+### To run locally without Docker:
+
+To run the application without Docker, follow these steps:
+
+- ***Clone the project repository from the version control system of your choice (e.g., Git) to your local machine.***
+    ```bash
+    git clone <repository-url>
```

Comment:
you have exact url to clone the repo, IMO placeholder can be replaced here by real link